packages/main/src/webComponents/UploadCollection/index.tsx

```diff
@@ -42,17 +43,17 @@ export interface UploadCollectionPropTypes extends Omit<WithWebComponentPropType
    *
    * **Note:** The `drop` event is fired only when elements are dropped within the drag and drop overlay and ignored for the other parts of the `UploadCollection`.
    */
-  onDrop?: (event: CustomEvent<{ dataTransfer: DataTransfer }>) => void;
+  onDrop?: DragEventHandler<HTMLElement>;
```

Comment:
Does `dataTransfer` not exist anymore, or was this forgotten with the refactoring?

Reply:
`dataTransfer` is defined by the `DragEventHandler`, having that as part of the `detail` is actually wrong